pkg/provider/github/github.go

```diff
@@ -238,13 +239,35 @@ func (w *GitHub) HandleHook(ctx context.Context, req *http.Request) (int, error)
 func (w *GitHub) handleEvent(ctx context.Context, client *github.Client, event interface{}, receiver *webhookv1.GitWatcher) (int, error) {
 	execution := initExecution(receiver)
 	switch event.(type) {
+	case *github.CreateEvent:
+		if receiver.Spec.Tag == false {
+			return http.StatusUnprocessableEntity, fmt.Errorf("create events only support tag")
```

Comment:
Not likely to happen. (if tag is false you won't get create event) Can we change WatchOnTag is not turned on?

Reply:
updated